### PR TITLE
Provide inspector indicator on destroyed services.

### LINKED ISSUE #1503: When a user clicks destroy in the inspector we should "mark" the service to be destroyed and provide a way to "un-do"
This is instead of showing the "cancel" and "confirm" buttons.

blocked-service2

### PR DESCRIPTION
**Pull request by kadams54**
Fixes #1503.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5925/
Test FAILed.

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5932/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5936/
Test PASSed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5937/
Test PASSed.

**Comment by kadams54** [comment]
## QA

1. `make run`
2. Head to http://0.0.0.0:6543/.
3. Add an application to the canvas and deploy it.
4. Select the application to open the inspector.
5. Click on the "Destroy" button.
6. Verify that the delete command goes into pending changes. The app icon on the canvas should turn light blue, the delete button should disappear, and a message should appear in the inspector about the pending deletion, similar to the image below. The "Cancel" button will not be there, as the ability to "undo" a specific pending change is not yet supported.

blocked-service2

File: jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js [hunk 1]
```diff
@@ -311,52 +318,44 @@ YUI.add('service-overview', function() {
...
-        action: this._hideConfirmation
-      }, {
-        disabled: disabled,
-        title: 'Confirm',
-        type: 'destructive',
-        action: this._destroyService
-      }];
+      const confirmationOpen = this.state.confirmationOpen;
+      const readOnly = this.props.acl.isReadOnly();
+      const message = 'This service has been marked to be destroyed on next '
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/inspector/service-overview/service-overview.js]: s/service/application

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5938/
Test FAILed.

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5939/
Test PASSed.

**Comment by kadams54** [comment]
Thanks @hatched! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/5940/
Test PASSed.

**Pull request merged**